mlir/lib/Dialect/Vector/Transforms/LowerVectorBroadcast.cpp

```diff
@@ -85,7 +85,8 @@ class BroadcastOpLowering : public OpRewritePattern<vector::BroadcastOp> {
     if (srcRank < dstRank) {
       // Duplication.
       VectorType resType =
-          VectorType::get(dstType.getShape().drop_front(), eltType);
+          VectorType::get(dstType.getShape().drop_front(), eltType,
+                          dstType.getScalableDims().drop_front());
```

Comment:
I noticed @MacDue has been doing:
```
VectorType::Builder(dstType).dropDim(0));
```

in his patches, which seems a bit nicer?

Reply:
Yep, that'd be preferred here :+1:

Reply:
Done, thanks!